site/content/internal/infrastructure/kubernetes/kubernetes-troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Kubernetes Maintenance"
+date: 2018-11-07T15:24:42+01:00
+weight: 10
+---
+
+This page helps developers access and perform any type of maintenance in the Production Mattermost Kubernetes Cluster, which is running on AWS using EKS.
+
+## Set up a local environment to access Kubernetes (K8s)
+
+1. Make sure you have `kubectl` version 1.10 or later installed. If not, follow [these instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+2. Use your OneLogin account to retrieve AWS Keys for the main Mattermost AWS account following [these instructions](../../onelogin-aws) to install onelogin-aws command line application.
+
+Steps:
```

Comment:
I'm not quite sure what this section is for, and it looks similar to the feedback in https://github.com/mattermost/mattermost-developer-documentation/pull/185#pullrequestreview-189798494

Are the steps for 2, in addition to the linked instructions?

Reply:
yes